### ISSUE TITLE
Organization. update

### ISSUE THREAD

**Issue by r-peschke** [OpenSlides/openslides-backend#864]
update organization's name has to be possible with OML-right. Currently an error message demand superadmin right

**Comment by jsangmeister** [comment]
Can you provide a payload where this error occurs? It is currently implemented correctly. I suspect that you also sent one of the fields `enable_electronic_voting` or `reset_password_verbose_errors` and that's where the error stems from.

**Comment by r-peschke** [comment]
I wrote it on behalf of Emanuel and will ask him.

**Comment by emanuelschuetze** [comment]
@jsangmeister You're right. The client (with orgaadmin) sends also `enable_electronic_voting` and `reset_password_verbose_errors` in payload:

{
    "name": "OS4-Demo 1",
    "description": "",
    "legal_notice": "<p><a href=\"http://www.openslides.org\">OpenSlides</a> is a free web based presentation and assembly system for visualizing and controlling agenda, motions and elections of an assembly. The event organizer is resposible for the content.</p>",
    "privacy_policy": "",
    "login_text": "<p>Demo-Login:<br>Benutzername: <b>admin</b> · Passwort: <b>admin</b><p>",
    "theme": "openslides-default-light-theme",
    "enable_electronic_voting": true,
    "reset_password_verbose_errors": false,
    "id": 1
}

**Comment by jsangmeister** [comment]
Then this is a client issue, the values should be remove from the payload before sending it (or rather, the input should not event exists in the first place).
